pom.xml

```diff
@@ -185,11 +167,6 @@
                 <artifactId>mockito-core</artifactId>
                 <version>${global_mockito_version}</version>
             </dependency>
-            <dependency>
```

Comment:
I don't understand why `storm-kafka` was defined as a dependency here originally.  After removing it, I compared all the dependencies that maven reports via `dependency:tree` and nothing changed.